java/core/page-factory-core/src/test/java/ru/sbtqa/tag/pagefactory/pages/webelements/IndexPage.java

```diff
@@ -3,19 +3,32 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import ru.sbtqa.tag.pagefactory.PageFactory;
+import ru.sbtqa.tag.pagefactory.annotations.ActionTitle;
 import ru.sbtqa.tag.pagefactory.annotations.ElementTitle;
 import ru.sbtqa.tag.pagefactory.annotations.PageEntry;
+import ru.sbtqa.tag.pagefactory.annotations.RedirectsTo;
 
 @PageEntry(title = "Main")
 public class IndexPage extends AbstractPage {
     
     @ElementTitle("Sample text")
     @FindBy(xpath = "//*[@id='samples']/p[@class='lead']")
     private WebElement sampleText;
-    
+
     @ElementTitle("License text")
     @FindBy(xpath = "//*[@id='license']/p[@class='lead']")
     private WebElement licenseText;
+
+    @ActionTitle("go to page from Main")
+    public void goToPage(String pageName) {
+        switch (pageName.toLowerCase()) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1301)